### DISCUSSION TITLE
I keep getting rejected by incorrect billing's name

### DISCUSSION THREAD

**Discussion by haltinganhdeptrai** [community/community#146240] [GitHub Education]
(sorry for my bad English)
Hi, 
My name is Vũ Minh Tiến (vietnamese) and it's also the name that appear on my student id's card.

My country citizen's name has different display compare to other foreign countries which my name display is:
Last name: Vũ
Middle name: Minh
First name: Tiến
Full: Vũ Minh Tiến

But when I fill in my billing information, it ask for my first and last name but when it display it out, my billing information turn into "Tiến Vũ" (which I believe follow foreign countries display order) 
That make (atleast what I think) my application being rejected because it say what I input doesn't right and it has to be display completely match with what is shows on my student id's card

So I try different display such as
Last name: Vũ
First name: Minh Tiến
Or the opposite way to let it display as "Vũ Minh Tiến" but it still doesn't work

Can someone help?
Thanks for reading!!
[Screenshot_2024-12-04-17-32-30-899_com.google.android.gm.jpg](https://github.com/user-attachments/assets/77c6b68f-dacb-43d2-8ee9-373a0b1eb3ed)

**Comment by ahmedb500** [comment]
Can you help me understand the problem more please explain

**Comment by Sheikh-Anas-Tauseef** [comment]
Hi Tiến 👋,

Thanks for sharing your issue. You're correct — Vietnamese names follow the order: Last Name → Middle Name → First Name, while many international systems use First Name → Last Name, which can cause problems with verification.

Here’s a suggestion that might help:
When entering your billing info:

First Name: Minh Tiến

Last Name: Vũ

This way, the display will match your full name: Vũ Minh Tiến, exactly as it appears on your student ID.

If it still doesn’t work, I recommend:

Contacting GitHub or the billing system's support.

Explaining the cultural name order difference.

Attaching a photo of your student ID for verification.

✨ This is a common issue with non-Western name formats, and support teams are usually understanding if you explain it clearly.
